examples/nlp_seq/train.py

```diff
@@ -301,7 +294,7 @@ def main(argv):
   @jax.jit
   def initialize_variables(init_rng):
     init_batch = jnp.ones((config.max_len, 1), jnp.float32)
-    init_variables = model.init(init_rng, init_batch)
+    init_variables = model.init(init_rng, inputs=init_batch, train=False)
```

Comment:
after my proposed change in `models.py`, you can revert here to

```
init_variables = model.init(init_rng, init_batch, train=True
```

(we usually say `train=True` for initialization)

Reply:
I changed this see, see new commit.